homeassistant/components/media_player/plex.py

```diff
@@ -196,16 +195,16 @@ def supported_media_commands(self):
 
     def media_play(self):
         """ media_play media player. """
-        self.device.play({'type': 'video'})
+        self.device.play()
```

Comment:
In the 'regular' plexapi, these functions do not support parameters.

Playback of music is not detected, so the media player looks 'idle' in HA.
Playback controls for music work fine though.